packages/flutter_tools/test/commands.shard/permeable/upgrade_test.dart

```diff
@@ -44,10 +44,15 @@ void main() {
       fakePlatform = FakePlatform()..environment = Map<String, String>.unmodifiable(<String, String>{
         'ENV1': 'irrelevant',
         'ENV2': 'irrelevant',
-      });
+      })..version = '1.2.3-dev-4.5';
```

Comment:
I don't think this is necessary? The tests pass without it. And I'm not familiar the `version` property of `Platform`.

Reply:
Yes, it isn't. Probably a leftover I overlooked. I removed it.